src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php

```diff
@@ -73,22 +73,16 @@ public function normalize($object, $format = null, array $context = array())
                 $attributeValue = call_user_func($this->callbacks[$attribute], $attributeValue);
             }
 
+            // Recursive call are done at the end of the process to allow @MaxDepth to work
```

Comment:
`Recursive call are done` -> `Recursive call is done` or `Recursive calls are done`